### PR TITLE
Story chunks and PlayerTurnSubmit v2

### PR DESCRIPTION
**Pull request by werka-z**
Fixed [problems from v1](https://github.com/QuestUJ/QuestAssembler/pull/30), added some unit tests, altered CharacterDetails logic a bit as discussed.

### CODE REVIEW

File: server/src/domain/game/Room.ts [hunk 1]
```diff
@@ -26,6 +26,9 @@ export class RoomSettings {
 }
 
 export class Room {
+    gameMaster(gameMaster: any) {
```
Comment (St0pien) [inline server/src/domain/game/Room.ts:29]: I don't think it's necessary, we already have `getGameMaster` getter

File: server/src/domain/game/StoryChunk.ts [hunk 2]
```diff
@@ -1,9 +1,15 @@
 export class StoryChunk {
-    constructor(
-        readonly id: number = -1,
-        readonly title: string = '',
-        readonly content: string = '',
-        readonly imageUrl: string = '',
-        readonly timestamp: Date = new Date()
-    ) {}
+    readonly id: number;
```
Comment (St0pien) [inline server/src/domain/game/StoryChunk.ts:2]: Isn't it the same but with more lines?

**Review by St0pien — CHANGES_REQUESTED** [review]
Well done! please refer to comments

File: server/src/domain/game/StoryChunk.ts (hunk 2 above)
Comment (werka-z) [reply server/src/domain/game/StoryChunk.ts:2]: I wanted to initialize only the title for a test, but sure, I can leave this constructor and initialize both the id and title.

File: server/src/domain/game/Room.ts (hunk 1 above)
Comment (werka-z) [reply server/src/domain/game/Room.ts:29]: yes, I don't even recall writing this so this must have been a linter auto-fix or something😅

**Review by St0pien — APPROVED** [review]

**Pull request merged**